include/aspect/simulator_access.h

```diff
@@ -781,14 +781,32 @@ namespace aspect
        * constraints for the time step we are currently solving.
        */
       const ConstraintMatrix &
-      get_current_constraints() const;
+      get_current_constraints () const;
 
       /**
        * Return whether or not the current object has been initialized by providing it with
        * a pointer to a Simulator class object.
        */
       bool simulator_is_initialized () const;
 
+      /**
+       * Return whether the Simulator object has been completely initialized
+       * and has started to run its time stepping loop. Obviously, for this
+       * function to work, the simulator_is_initialized() function needs
+       * to return @p true.
+       *
+       * This function is useful to determine in a plugin whether some
+       * of the information one can query about the Simulator is reliable
```

Comment:
what is the difference between "reliable" or "has already been set"? I find the word "reliable" confusing. Maybe state that initialization of all plugins and settings is completed? I guess the main difference between ``simulator_is_initialized`` and this function is that the first only says that initialization started and some information is not valid/set yet.